ClientSideEncryptionToCPKN/ClientSideKeyVaultKeyToCustomerManagedKey/samplesrc/Migration.cs

```diff
@@ -0,0 +1,130 @@
+﻿using Azure.Storage.Blobs;
+using Azure.Storage.Blobs.Models;
+using Azure.Storage.Blobs.Specialized;
+using Azure.Storage;
+using Azure.Core;
+using Azure.Security.KeyVault.Keys.Cryptography;
+using Azure.Identity;
+using System;
+using System.IO;
+
+
+namespace keyVaultClientSideToCustomerManagedServerSide
+{
+    class Migration
+    {   
+        //Downloads and decrypts client side encrypted blob, then reuploads blob with server side encryption using an encryption scope with a customer managed key
+        private static void EncryptWithCustomerManagedKey(
+            string connectionString,
+            string containerName,
+            string fileName,
+            string filePath,
+            ClientSideEncryptionOptions clientSideOption,
+            string encryptionScopeName)
+        {
+            //Download and decrypt Client Side Encrypted blob using BlobClient with Client Side Encryption Options
+            string downloadFilePath = filePath.Replace(".txt", "Download.txt");
```

Comment:
this won't work for non '.txt' files. I'd rather just add suffix `.download` or something like that. We should assume that customer can have any extension (or even none)